app/assets/javascripts/action_tracker.js

```diff
@@ -2,27 +2,37 @@
 var ActionTracker = function () {
 
   var storage,
+      time_seed,
+      options = {},
       callbacks = {};
 
   function constructor() {
     storage = new Storage();
+    time_seed = new TimeSeed();
   }
 
   function setCallbacks(callbacksObj) {
     callbacks = callbacksObj;
   }
 
-  function start(list) {
+  function start(list, cfg_options) {
+    if(typeof cfg_options !== 'undefined') options = cfg_options;
     storage.queue(list);
   }
 
   function process() {
     while(typeof storage.getFirst() !== 'undefined') {
-      var tracker = new Tracker(storage.dequeue());
-      tracker.send();
+      var tracker = new Tracker(storage.dequeue(), trackerOptions());
+      tracker.addTimestamp(time_seed).send();
     }
   }
 
+  function trackerOptions() {
+    return {
+      timestamp: (typeof options.timestamp !== 'undefined') ? options.timestamp : true
```

Comment:
Why ```true``` is the default option? If this behavior is not related to core, I think it should be false, then if we want to use it, set it. Isn't it right? Will it change the interface?